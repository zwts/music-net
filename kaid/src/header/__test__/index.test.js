import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../index';

describe('<Header />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Header>KaiOS header</Header>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
