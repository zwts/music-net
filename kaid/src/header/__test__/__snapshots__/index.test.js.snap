// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<header
  className="kai-header"
>
  <h1
    className="h1"
  >
    KaiOS header
  </h1>
</header>
`;
